ack/readers/awin_advertiser/config.py

```diff
@@ -39,7 +39,10 @@ class AwinAdvertiserReaderConfig(BaseModel):
     advertiser_id: str
     report_type: Literal[tuple(REPORT_TYPES)]
     region: str
+    campaign: str = None
     timezone: str
+    interval: str = None
+    remove_tags: str = 'False'
```

Comment:
remove_tags should be a boolean with a default value at False (without quotes as it's a real boolean)